lib/spectrum/request/place_hold.rb

```diff
@@ -90,10 +90,11 @@ def fetch_bib_record
       end
 
       def fetch_holdings_record
-        uri = URI(@source.holdings + @record_id)
-        holdings_data = JSON.parse(Net::HTTP.get(uri))
-        barcode = get_barcode_from_holdings_data_with_key(holdings_data, @item_key)
-        Spectrum::Item.for_barcode(holdings_data[@record_id], @record_id, barcode)
+        req = Object.new
+        def req.id; @record_id; end
```

Comment:
I'm not as used to seeing this pattern.  That said I'm okay with it, though I generally prefer something like: 
```
req = Struct.new(:id).new(@record_id)

# or

OpenStruct.new(id: @record_id)
```
Anyway, there's no need to make a change, unless you like the alternatives better.